redact/__main__.py

```diff
@@ -8,19 +8,46 @@
 from redact.tools.redact_folder import redact_folder as rdct_folder
 from redact.tools.redact_folder import InputType
 
-
 settings = Settings()
 
 
-def redact_file(file_path: str, out_type: OutputType, service: ServiceType, region: Region = Region.european_union,
-                face: bool = True, license_plate: bool = True,
-                licence_plate_custom_stamp_path: Optional[str] = typer.Option(None, '--custom-lp', help='Image file to use for license plate replacements'),
-                custom_labels_file_path: Optional[str] = typer.Option(None, '--labels', help='A JSON file containing custom labels'),
-                redact_url: str = settings.redact_url_default, api_key: Optional[str] = None,
+def redact_file(file_path: str,
+                out_type: OutputType,
+                service: ServiceType,
```

Comment:
why do these not have a `typer.Option`? Looks like you added them and then removed them again.